types/terminal.d.ts

```diff
@@ -91,9 +91,31 @@ export declare type TerminalProps = TerminalOptions & TerminalCallbacks;
 
 export declare type PaymentIntentClientSecret = string;
 
-export declare type DeviceType = Stripe.Terminal.Reader.DeviceType;
-
-export declare type Reader = Stripe.Terminal.Reader;
+export enum DeviceType {
+  P400 = 'verifone_P400',
+  WISEPOSE = 'bbpos_wisepos_e',
+  S700 = 'stripe_s700',
```

Comment:
in order to get S700 i need to bump the API version quite a bit and I don't really know the breaking change implications of that right now. JS SDK itself maintains its own type so seems better to use the same here.

Reply:
oh interesting so this will be hardcoded temporarily(?) is there going to be a longer-term fix?